tests/test_bisnd_mcc.py

```diff
@@ -54,7 +54,7 @@ def test_status():
 
 def test_location():
     assert parsed_item["location"] == {
-        "address": "Morton County Commission Room",
+        "address": "Morton County Commission Room",  # noqa shouldn't "address" value contain the street/city/state instead of "name" value?
```

Comment:
**praise (blocking):** I think you're totally right. Thanks for catching that. If you can remove this comment I'll address this in a separate PR. In future, I'd suggest flagging issues like these as a Github issue or a PR note just so we can keep the commit history of a PR clean and focused on a specific feature/fix, but this is no big deal. Thanks again. I appreciate you catching this 👍

Reply:
Gotcha. Wasn't aware of the procedure. Will clean up. Thanks!